scss/components/_masonry-grid.scss

```diff
@@ -0,0 +1,4 @@
+.civicase__masonry-grid__container {
```

Comment:
it would be better to have the parent to apply any layout-related style
```scss
[civicase-masonry-grid] {
  .civicase__masonry-grid__container {
    float: left;
    width: 50%;
  }
}